src/Build/BackEnd/Components/ProjectCache/ProjectCacheService.cs

```diff
@@ -525,7 +525,7 @@ private async Task<CacheResult> GetCacheResultAsync(BuildRequestData buildReques
 
                     ErrorUtilities.VerifyThrow(plugin.Instance != null, "Plugin '{0}' instance is null", plugin.Name);
 
-                    MSBuildEventSource.Log.ProjectCacheGetCacheResultStart(plugin.Name, buildRequest.ProjectFullPath, targetNames);
+                    MSBuildEventSource.Log.ProjectCacheGetCacheResultStart(plugin.Name, buildRequest.ProjectFullPath, targetNames ?? "<default>");
```

Comment:
possibly future nit:
localize?

It'd be higher priority, but I don't think non-experts use this very much, so it might not matter.

Reply:
IMO the ETW doesn't need localization, and even if it did--the targets don't have localized names, so this doesn't seem unreasonable.